src/modules/Player/Components/PlayButton/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+type Props = {
+  playerState: boolean;
+  setPlayerState: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+const PlayButton: React.FC<Props> = React.memo(({playerState, setPlayerState}) => {
+
+  const Play = () => {
+    setPlayerState(true);
+  };
+
+  const Pause = () => {
+    setPlayerState(false);
+  };
+
```

Comment:
```suggestion
const handleChange = () => setPlayerState(prev => !prev)
```

Вместо этих двух функций